logback-contrib/pom.xml

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2016, The logback-contrib developers. All rights reserved.
+
+    This program and the accompanying materials are dual-licensed under
+    either the terms of the Eclipse Public License v1.0 as published by
+    the Eclipse Foundation
+
+      or (per the licensee's choosing)
+
+    under the terms of the GNU Lesser General Public License version 2.1
+    as published by the Free Software Foundation.
```

Comment:
We shall relicense this module as **Eclipse Public License v1.0** only and we shall add a notice in `NOTICE.txt` at the root of the tmail-backend repo stating that *although twake mail is licenses as AGPL v3 we do ship a fork of logback contribs wothin XYZ folder subject to the Eclipse license 1.0*

Each file (xml / java) in this folder must have a comment stating:
 - the license: epl 1.0
 - where the original work can be found (github link is ok)